packages/shared/src/messages/MessageProxy.ts

```diff
@@ -78,12 +78,18 @@ export class RpcExtension {
           body: result,
           status: 'success',
         } as IMessageResponse);
-      } catch (e) {
+      } catch (err: unknown) {
+        let errorMessage: string;
+        if (err instanceof Error) errorMessage = err.message;
+        else if (typeof err === 'string') errorMessage = err;
+        else errorMessage = String(err);
```

Comment:
please add curly brackets so it's in line with the rest of the code

Reply:
Done